internal/store/search_nodes.go

```diff
@@ -21,12 +21,16 @@ import (
 	atlasv2 "go.mongodb.org/atlas-sdk/v20231115006/admin"
 )
 
-//go:generate mockgen -destination=../mocks/mock_search_nodes.go -package=mocks github.com/mongodb/mongodb-atlas-cli/internal/store SearchNodesLister
+//go:generate mockgen -destination=../mocks/mock_search_nodes.go -package=mocks github.com/mongodb/mongodb-atlas-cli/internal/store SearchNodesLister,SearchNodseCreator
```

Comment:
```suggestion
//go:generate mockgen -destination=../mocks/mock_search_nodes.go -package=mocks github.com/mongodb/mongodb-atlas-cli/internal/store SearchNodesLister, SearchNodesCreator
```

Reply:
@jeroenvervaeke i think i added an extra space in my suggestion accidentally FYI